data/DEDIS/projects.yaml

```diff
@@ -531,3 +531,37 @@ projects:
     license: other
     date_added: 2022-03-16
     date_updated: 2022-03-16
+
+  apir:
+    name: APIR
+    categories:
+      - Privacy
+    applications:
+      - Info
+    description: >
+      This work introduces protocols for authenticated private information retrieval, 
+      which enable a client to fetch a record from a database server such that (a) the server does 
+      not learn which record the client reads, and (b) the client either obtains the "authentic" 
+      record or detects server misbehavior and safely aborts.
+    contacts:
+      - name: Simone Colombo
+        email: simone.colombo@epfl.ch
+    tags:
+      - Anonymity
+      - Access Control
+    type: Library
+    code:
+      type: Lab GitHub
+      url: https://github.com/dedis/apir-code
+      date_last_commit: 2023-05-04
+    information:
+      - type: Paper
```

Comment:
Can you look if it exists on https://infoscience.epfl.ch and update the link to this one? But then you also need to check if it's publicly available on this URL, and if it's not, add a second link to the eprint.iacr.org, similar to this (which is for a published paper, but I think we can just add "eprint location"):

https://github.com/c4dt/showcase/blob/main/data/DEDIS/projects.yaml#L133

Reply:
It doesn't exist there. Simone has only one paper on infoscience. [reference](https://infoscience.epfl.ch/search?f1=author&as=1&sf=title&so=a&rm=&m1=e&p1=Colombo%2C%20Simone&ln=en)